guides/advanced_guides/faceted_search.md

```diff
@@ -24,14 +24,15 @@ Faceting and filtering aim at being complementary;  facets narrows down the set
 
 ## Setting Up Facets
 
-Attributes that could be used as facets **must be declared at indexing time**.
+The first step in using facets is to chose which of your document <clientGlossary word="field" label="fields"/> will be used as facets. Fields with common values are the most suited for faceting (e.g., `genre`, `color`, `size` ).
 
-You can set up facets **through the API** via the [global settings route](/references/settings.md#update-settings).
-Attributes for which to enable faceting are defined in the `attributesForFaceting` list.
+For these fields to be used as facets during search, their <clientGlossary word="attribute" label="attributes"/> **have to be previously added to the settings**. In the settings, the chosen attributes must be added to the [`attributesForFaceting` list](/guides/advanced_guides/settings.md#attributes-for-faceting).
+It is a required step because facet needs to be properly processed and prepared by the engine to be usable. This process takes as much time as indexing all your documents.
```

Comment:
```suggestion
This step is required because facet needs to be properly processed and prepared by the engine to be usable. This process takes as much time as indexing all your documents.
```